### PR TITLE
Implements https://github.com/appnexus/sicksync/issues/20

### PR DESCRIPTION
**Pull request by joelgriffith**
https://github.com/appnexus/sicksync/issues/20

Anytime sicksync starts, or gets disconnected, it'll do a `bigSync` before emitting web-socket changes.

### CODE REVIEW

File: lib/ws-client.js [hunk 1]
```diff
@@ -52,13 +53,20 @@ function handleDisconnect() {
 }
 
 function handleError() {
-    var message = [
-        'Starting syncer with ' + config.hostname.yellow
+    var starMessage = [
```
Comment (chrisdopuch) [inline lib/ws-client.js]: Typo on startMessage

**Comment by chrisdopuch** [comment]
Looks nice, +1

**Comment by joelgriffith** [comment]
I'll update docs/readme/relase once merged into master.

File: lib/ws-client.js (hunk 1 above)
Comment (joelgriffith) [inline lib/ws-client.js]: Spelling and I don't mix.

**Pull request merged**